app/index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html ng-app="MyApp">
+<head>
+    <title></title>
+    <link rel="stylesheet" type="text/css"
+          href="http://netdna.bootstrapcdn.com/twitter-bootstrap/2.2.2/css/bootstrap-combined.min.css"/>
+    <link rel="stylesheet" href="styles.css">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
```

Comment:
add jqery to `package.json` as dependency

Reply:
done